server/src/test/java/keywhiz/commands/BackfillOwnershipCommandIntegrationTest.java

```diff
@@ -13,16 +13,21 @@
 import java.util.UUID;
 import java.util.concurrent.TimeUnit;
 import javax.inject.Inject;
+import keywhiz.IntegrationTestRule;
 import keywhiz.KeywhizTestRunner;
 import keywhiz.test.ServiceContext;
 import net.sourceforge.argparse4j.inf.Namespace;
 import org.jooq.DSLContext;
 import org.jooq.Record1;
+import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.rules.RuleChain;
 import org.junit.runner.RunWith;
 
 @RunWith(KeywhizTestRunner.class)
-public class BackfillOwnershipCommandTest {
+public class BackfillOwnershipCommandIntegrationTest {
+  @ClassRule public static final RuleChain chain = IntegrationTestRule.rule();
```

Comment:
There were two tests (both backfills) which weren't properly configured to use/migrate the database and depended on execution order to succeed. Explicitly adding the integration test rule (which in turn runs Flyway's migration logic) fixed that issue. Also, since we only needed to touch two tests, I decided to rename them to be actual integration tests to be more clear (and note that there's another 150 or so which should get the same treatment, but that's a fight for another day).

Reply:
good catch!